doc/nrf/ug_multi_image.rst

```diff
@@ -182,6 +180,54 @@ This means that to run menuconfig, for example, you invoke the ``menuconfig`` ta
 You can also invoke any child target directly from its build directory.
 Child build directories are located at the root of the parent's build directory.
 
+Controlling the build process
+=============================
+
+The child image is built using CMake's built-in build command ``cmake --build``.
```

Comment:
Is it important that this is a "built-in" command? As compared to what?

Reply:
No, will remove `built-in`.